php/class-delivery.php

```diff
@@ -1016,11 +1016,13 @@ public function convert_tags( $content, $context = 'view' ) {
 				continue;
 			}
 			if ( ! empty( $relation['slashed'] ) && $relation['slashed'] ) {
-				$aliases[ $base . '?' ] = addcslashes( $cloudinary_url . '&', '/' );
-				$aliases[ $base ]       = addcslashes( $cloudinary_url, '/' );
+				$aliases[ $base . '?_i=AA' ] = addcslashes( $cloudinary_url, '/' );
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question**
is there any chance there would be more query variables? Adding more also causes the sitemap to fail.

Reply:
@tharsheblows 

Great finding.

My testing will depend on whether the `_i` argument is the first. If so, this will still work fine; otherwise, it will fail.

These should be edge cases; we might have better ways to work around them.

Let's follow up internally.